content/docs/iac/concepts/vs/cloud-template-transpilers/aws-cdk/_index.md

```diff
@@ -34,6 +34,41 @@ Choosing the right infrastructure as code tool is important, and we want you to
 
 AWS Cloud Development Kit (CDK) is an open-source software development framework for defining AWS cloud resources with general-purpose programming languages. CDK is a transpiler that converts program code into AWS CloudFormation JSON/YAML templates and other assets and then submits them to AWS and the CloudFormation deployment service.
 
+## Why Choose Pulumi over AWS CDK?
+
+Pulumi has various benefits over AWS CDK across dimensions of speed, security, savings, and scale.
+
+### Speed
+
+Pulumi enables faster development and deployment workflows compared to CDK due to its direct integration with cloud providers without going through a transpiler.
+
+- __Deployment__. Pulumi communicates directly with cloud providers, bypassing the intermediate step of generating CloudFormation templates as required by CDK. Since CDK relies on CloudFormation as its deployment engine, it inherits many of CloudFormation's speed limitations. (see [Pulumi vs. CloudFormation](/docs/concepts/vs/cloud-templates/cloudformation/)).
+- __Development__. Pulumi's runtime engine allows for immediate feedback during deployment, enabling developers to debug and iterate more quickly. In contrast, CDK's reliance on CloudFormation introduces delays due to template transpilation and longer error resolution times.
```

Comment:
We are stating this very "cold and factually"; the fact that Pulumi lets you find errors instantly in your IDE, in milliseconds, for things that with CloudFormation can take tens of minutes, is a _big deal_. We should be more matter of fact about this.